flink-python/pyflink/table/tests/test_table_environment_api.py

```diff
@@ -700,17 +700,17 @@ def test_collect_for_all_data_types(self):
                                1.98932, bytearray(b'pyflink'), 'pyflink',
                                datetime.date(2014, 9, 13), datetime.time(12, 0, 0, 123000),
                                datetime.datetime(2018, 3, 11, 3, 0, 0, 123000),
-                               [Row(['[pyflink]']), Row(['[pyflink]']), Row(['[pyflink]'])],
-                               {1: Row(['[flink]']), 2: Row(['[pyflink]'])},
+                               [Row('pyflink'), Row('pyflink'), Row('pyflink')],
```

Comment:
nit: I suggest some more complex tests , arrays of arrays, array of rows of arrays, and include maps and lists, just to make sure all cases are as expected.

Reply:
Good point, I'll add some more complex cases here